packages/devtools_app/lib/src/memory/memory_allocation_table_view.dart

```diff
@@ -234,7 +234,8 @@ class AllocationTableViewState extends State<AllocationTableView>
       axis: Axis.vertical,
       children: [
         controller.allocationsFieldsTable,
-        trackerData.createTrackingTable(context, controller, scrollerController),
+        trackerData.createTrackingTable(
+            context, controller, scrollerController),
```

Comment:
Don't know why the bot didn't show this, but should have an ending comma.

Reply:
done